internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-operator.go

```diff
@@ -150,7 +152,7 @@ func (tec *TunnelEndpointCreator) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 	// examine DeletionTimestamp to determine if object is under deletion
 	if netConfig.ObjectMeta.DeletionTimestamp.IsZero() {
-		if !liqonet.ContainsString(netConfig.ObjectMeta.Finalizers, tunnelEndpointCreatorFinalizer) {
+		if !liqonetutils.ContainsString(netConfig.ObjectMeta.Finalizers, tunnelEndpointCreatorFinalizer) {
```

Comment:
I suppose you could use the standard functions to manage finalizers from the [controllerutil package](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#ContainsFinalizer) instead of a custom implementation.

Same for all occurrences below